### PR TITLE
Onboard Apicurio on ArgoCD

### PR DESCRIPTION
**Pull request by paoloantinori**
Tracking feedback on the docs here:

https://github.com/operate-first/support/issues/197

### CODE REVIEW

File: argocd/overlays/moc-infra/projects/apicurio-registry.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
```
Comment (HumairAK) [inline argocd/overlays/moc-infra/projects/apicurio-registry.yaml:3]: Can you add a label:
```yaml
  labels:
    project-template: global
```

This should have been reflected in the docs (but isn't), I'll update docs accordingly.

**Comment by HumairAK** [comment]
Can you add the following line[ as per the docs](https://github.com/operate-first/support/blob/main/docs/onboarding_to_argocd.md#give-team-read-access-to-non-app-argocd-resources): 

`g, apicurio, role:standard-user` 

to this file: https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/configs/argo_rbac_cm/policy.csv#L21

File: argocd/overlays/moc-infra/projects/apicurio-registry.yaml (hunk 1 above)
Comment (paoloantinori) [reply argocd/overlays/moc-infra/projects/apicurio-registry.yaml:3]: done, thank you

**Comment by paoloantinori** [comment]
> Can you add the following line[ as per the docs](https://github.com/operate-first/support/blob/main/docs/onboarding_to_argocd.md#give-team-read-access-to-non-app-argocd-resources):
> 
> `g, apicurio, role:standard-user`
> 
> to this file: https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/configs/argo_rbac_cm/policy.csv#L21

done, I had definitely overlooked that. If I can offer a little suggestion, I would consider making that step a little more evident, possibly quoting the quote that needs to be added. It would help to having it sticking out more.

Thank you

**Review by tumido — COMMENTED** [review]
/lgtm

**Review by HumairAK — APPROVED** [review]

**Comment by HumairAK** [comment]
Changes were applied @paoloantinori let us know if you can log in to argocd and create projects: https://argocd.operate-first.cloud 

You can create projects direclty via the ui, though these will not be backed up. To ensure your application is restored during clusterupgrades/restarts, we suggest adding it in version control here: https://github.com/operate-first/argocd-apps 
Using: https://github.com/operate-first/argocd-apps/blob/main/docs/add_application.md

**Pull request merged**